src/main/java/school/lemon/changerequest/java/generics/NumberGenerator/SumCalculatorInteger.java

```diff
@@ -0,0 +1,17 @@
+package school.lemon.changerequest.java.generics.NumberGenerator;
+
+/**
+ * Created by lera on 07.01.17.
+ */
+public class SumCalculatorInteger extends SumCalculator<Integer> {
+
+    @Override
+    public Integer calculateSum(Integer[] array) {
+        Integer sum = new Integer(0);
```

Comment:
No need to use `new` here.
For Integer, Double, Float etc it can be done like `Integer sum = 0;`. Please do it this way.
The same for double calculator.